libraries/chain/fork_database.cpp

```diff
@@ -666,15 +670,15 @@ namespace eosio::chain {
 
    template<class BSP>
    BSP fork_database_t<BSP>::get_block(const block_id_type& id,
-                                       check_root_t check_root /* = check_root_t::no */) const {
+                                       include_root_t include_root /* = include_root_t::no */) const {
       std::lock_guard g( my->mtx );
-      return my->get_block_impl(id, check_root);
+      return my->get_block_impl(id, include_root);
    }
 
    template<class BSP>
    BSP fork_database_impl<BSP>::get_block_impl(const block_id_type& id,
-                                               check_root_t check_root /* = check_root_t::no */) const {
-      if( check_root == check_root_t::yes && root->id() == id ) {
+                                               include_root_t include_root /* = include_root_t::no */) const {
```

Comment:
Remove the /* = include_root_t::no */

Reply:
Why? It provides doc on what the default is.